### PR TITLE
Social Auth: Update TOS message to remove mention of creating an account on social login

### PR DESCRIPTION
**Pull request by zaguiini**
### Proposed Changes

On the login page, we warn the user that we're creating them an account if it doesn't exist when they try to socially sign in:

But that's not true: we don't try to, and warn them that they should sign up instead (notice on top). This is misleading, so we need to update the TOS. Also, a legal representative asked to differentiate the link color from the text: p2y3YZ-5Oq-p2#comment-13737.

### Testing Instructions

Check that the login and sign-up pages do not suggest that we're creating a new account in case social login fails. It also puts the ToS copy at the beginning and covers all three authentication methods.

Don't forget to run Calypso locally. Copy the link and open it in incognito mode.

| [`/log-in`](http://calypso.localhost:3000/log-in) | [`/log-in/new`](http://calypso.localhost:3000/log-in/new) |
| ---------| -------------- |
| image | image |

| [`/log-in/jetpack`](http://calypso.localhost:3000/log-in/jetpack) | [`/log-in?from=woocommerce-onboarding`](http://calypso.localhost:3000/log-in?from=woocommerce-onboarding) |
| ----------------- | ------------------------------------------- |
| image | image |

| [`/log-in?from=p2`](http://calypso.localhost:3000/log-in?from=p2) | [`/start/user`](http://calypso.localhost:3000/start/user) |
| ------------------ | ------------- |
| image | image |

| [Jetpack Connect](http://calypso.localhost:3000/log-in?client_id=69040&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dtoken%26client_id%3D69040%26redirect_uri%3Dhttps%253A%252F%252Fcloud.jetpack.com%252Fconnect%252Foauth%252Ftoken%253Fnext%253D%25252F%26scope%3Dglobal%26blog_id%3D0) | [WooCommerce connect](http://calypso.localhost:3000/log-in?client_id=50916&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3Db0e89241e21333a181c7611c1a247a628c141718e3e8e8ca6cbb432f522a7f3e%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253Dmy-dashboard%26blog_id%3D0%26wpcom_connect%3D1%26authorize%3D1%26wccom-from%3D%26calypso_env%3Dproduction) | [Crowdsignal connect](http://calypso.localhost:3000/log-in?client_id=978&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fclient_id%3D978%26response_type%3Dcode%26blog_id%3D0%26state%3D8aab737dadcdb664bc557210a1dba62242edaf47729eb82f0101765b13c57dad%26redirect_uri%3Dhttps%253A%252F%252Fapp.crowdsignal.com%252Fconnect%253Fsource%253Dlogin%2526action%253Drequest_access_token%26wpcom_connect%3D1) |
| ----------------- | ----------------- | ------------------ |
| image | image | image |

### CODE REVIEW

**Comment by zaguiini** [comment]
Working with a representative in p2y3YZ-5Oq-p2#comment-13724.

File: client/blocks/login/social.jsx [hunk 1]
```diff
@@ -140,72 +137,6 @@ class SocialLoginForm extends Component {
...
-						components: {
-							tosLink,
-							privacyLink,
-						},
-					}
-				) }
-			</p>
-		);
-	};
-
```
Comment (zaguiini) [inline client/blocks/login/social.jsx]: Now that we're showing a single ToS on the login page above the "Continue" button, there's no point in having this in the component as well. 

It doesn't affect the sign-up page, which has its own social component called [`SocialSignupForm`](https://github.com/Automattic/wp-calypso/tree/trunk/client/blocks/signup-form/social.jsx).

**Comment by danielbachhuber** [comment]
Added the `[Status] String Freeze` label so we can make sure the strings are translated before deploying.

**Review by danielbachhuber — APPROVED** [review]
Works great 👍 

Nice work tracking down all of the login variations!

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7507490

Thank you @zaguiini for including a screenshot in the description! This is really helpful for our translators.

**Comment by enejb** [comment]
Nice work with the changes @zaguiini. 

I found one more login case 
Screen Shot 2022-09-01 at 2 49 26 PM
[Woo login](http://calypso.localhost:3000/log-in?client_id=50916&redirect_to=https%3A%2F%2Fpublic-api.wordpress.com%2Foauth2%2Fauthorize%3Fresponse_type%3Dcode%26client_id%3D50916%26state%3D25afb3397ef89f804ab76f3b6a685235a22495221e9b54a6188391a2c8ce1c74%26redirect_uri%3Dhttps%253A%252F%252Fwoocommerce.com%252Fwc-api%252Fwpcom-signin%253Fnext%253Dmy-dashboard%26blog_id%3D0%26wpcom_connect%3D1%26authorize%3D1%26wccom-from%3D%26calypso_env%3Dproduction )

You get to it from woocommerace.com. 

Which looks correct. 

I found this page 
Screen Shot 2022-09-01 at 2 54 11 PM

[from woocommerce-onboarding](http://calypso.localhost:3000/log-in?from=woocommerce-onboarding) to be quite broken but I am not sure how it is used. 
But the broken things are not new in this PR :( - We should fix this in a different PR. (Not a blocker for this change)
it looks to me like the header shouldn't be there at all but it is. Also the styles seem really off.

**Review by enejb — APPROVED** [review]
Nice work!

**Comment by zaguiini** [comment]
Thanks, @enejb! Asked this in the Woo channel on Slack: p1662149049437139-slack-C07418EA0

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7507490).

**Pull request merged**